rules/id/id-jawi.js

```diff
@@ -0,0 +1,50 @@
+﻿( function ( $ ) {
+	'use strict';
+	var jawiKeyboard = {
+		id: 'id-jawi',
+		name: 'Jawi keyboard',
+		description: 'Jawi keyboard for Indonesian Wikisource',
+		date: '2023-09-21',
+		URL: 'https://www.mediawiki.org/wiki/Help:Extension:UniversalLanguageSelector/Input_methods/jawi-keyboard',
+		author: 'Benny Lin, Pungkas, Michael',
+		license: 'GPLv3',
+		version: '1.0',
+		contextLength: 3,
+		maxKeyLength: 3,
+		patterns: [
+			[ '\\\\([A-Za-z\\>_~\\.0-9])', '$1' ], //
```

Comment:
empty comment? I'm not sure what this line does. Identity transform for `\` followed by a letter? Shouldn't `$1` be a third parameter.

Reply:
changed. thanks.